dev.js

```diff
@@ -26,6 +27,7 @@ serverConfig.plugins.push(new webpack.BannerPlugin(
 ));
 
 var serverBundlePath = path.join(dirs.assets, 'server.bundle.js');
+var serverBundleRequirePath = isWin ? serverBundlePath.replace(/\\/g, '\\\\') : serverBundlePath;
```

Comment:
Right, that was an unnecessary check. I've removed it in the latest push. Btw, stay tuned for more windows issues coming up :)